lib/new_relic/agent/new_relic_service.rb

```diff
@@ -290,7 +291,11 @@ def setup_connection_for_ssl(conn)
         # installed
         conn.use_ssl     = true
         conn.verify_mode = OpenSSL::SSL::VERIFY_PEER
-        conn.cert_store  = ssl_cert_store
+        if @use_bundled_certs || NewRelic::Agent.config[:ca_bundle_path]
+          conn.cert_store  = ssl_cert_store
+        else
+          ::NewRelic::Agent.logger.debug("Using default security certificates")
```

Comment:
If I understand it correctly, we don't need to do that because ruby magic will just do it now with default stuff? Michael and I discussed it last week so I don't remember exactly now, but he might know. 
Also, it did work when skipping it when I was running it locally.

Reply:
Okay, sounds like this is a good thing to discuss as a group just to make sure everyone is clear on it. Thanks.